pkg/devfile/adapters/common/types.go

```diff
@@ -43,6 +43,7 @@ type PushParameters struct {
 	EnvSpecificInfo          envinfo.EnvSpecificInfo // EnvSpecificInfo contains infomation of env.yaml file
 	Debug                    bool                    // Runs the component in debug mode
 	DebugPort                int                     // Port used for remote debugging
+	ModeChanged              bool                    // It determines if mode is changed from run to debug or vice versa
```

Comment:
I think this should be descriptive like `RunModeChanged`

Reply:
Done